tools/lxdclient/client.go

```diff
@@ -52,11 +56,7 @@ var lxdLoadConfig = lxd.LoadConfig
 func newRawClient(remote, configDir string) (*lxd.Client, error) {
 	logger.Debugf("loading LXD client config from %q", configDir)
 
-	// This will go away once LoadConfig takes a dirname argument.
-	origDirname := updateLXDVars(configDir)
-	defer updateLXDVars(origDirname)
-
-	cfg, err := lxdLoadConfig()
+	cfg, err := lxdLoadConfig(path.Join(configDir, "config.yml"))
```

Comment:
Sure, but I don't know where/how to store it in juju's database, so I just left it as-is.